docs/manual/inference.tex

```diff
@@ -316,6 +316,14 @@
   is motivated by
   \ahref{https://github.com/gradle/gradle/issues/1319}{Gradle issue \#1319}.
 
+\item[-s]
+  If this flag is present, then projects which are not buildable---for which
+  no supported build file is present or for whom running the standard build
+  commands fail---are skipped on future runs but are NOT deleted immediately
+  (which is the default behavior for non-buildable projects). This flag is useful
```

Comment:
Rather than "default behavior", say, "if this flag is not present".  That is more explicit.